tools/DOCKERFILE

```
@@ -0,0 +1,22 @@
+# ベースイメージとしてAlpine Linuxを使用（軽量で高速）
+FROM alpine:latest
```

Hadolint

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)